### ISSUE TITLE
[Important accessibility improvement] Focus management on route transitions

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#2630]
[This is a ticket for the "accessibility backlog"]

When people use Single Page Applications just like Phoenix with keyboard-only or screen readers they need focus management after successful route change. Focus should be sent to the newly loaded content. This is especially important to screen reader users: When they interact with a route-link the route gets changed which is visually traceable. But for those screen reader users, they just stay on the link that they have just activated in silence.

Here's a suggestion on how to solve that in my project accessible-app.com: https://accessible-app.com/pattern/vue/routing

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
